fhir-server-test/src/test/java/com/ibm/fhir/server/test/SearchRevIncludeTest.java

```diff
@@ -7,6 +7,7 @@
 package com.ibm.fhir.server.test;
 
 import static com.ibm.fhir.model.type.String.of;
+import static org.junit.Assert.assertNull;
```

Comment:
per the prior coment ocnvert to testng's version of AssertJunit

Reply:
Done.